spec/mailers/publishers/expired_vacancy_feedback_prompt_mailer_spec.rb

```diff
@@ -36,5 +36,17 @@
     it "triggers a `publisher_prompt_for_feedback` email event" do
       expect { mail.deliver_now }.to have_triggered_event(:publisher_prompt_for_feedback).with_data(expected_data)
     end
+
+    context "from Sandbox environment" do
+      let(:notify_template) { NOTIFY_SANDBOX_TEMPLATE }
+
+      before do
```

Comment:
very minor thing, these could be one-liners:

`before { allow(Rails.env).to receive(:sandbox?).and_return(true) }`